lib/middleware/authorize.js

```diff
@@ -0,0 +1,10 @@
+module.exports = async (req, res, next) => {
+  try {
+    if (!req.user)
```

Comment:
Is there a situation in which a user would make it through the authenticate middleware(ensuring there is a user), but then wouldn't pass authorize? It looks like right now, any logged in user could make their way through this middleware. We can get really specific in this conditional, about the req.user.email - as an idea!